### PR TITLE
[PDMO-87]-Enable Retrieving of voided Observations

### PR DESCRIPTION
**Pull request by Farhanisto**
## JIRA TICKET NAME:

[PDMO-87 Enable Retrieving of voided Observations](https://issues.openmrs.org/browse/PDMO-87)

## SUMMARY:
Currently the voided observations can not be retrieved. In this ticket we will add functionality to enable retrieving and undeleting of the voided observations .

### CODE REVIEW

File: app/js/components/encounters/encountersParent.jsx [hunk 1]
```diff
@@ -92,15 +92,16 @@ export default class Encounters extends React.Component {
       })
       .catch(error => toastr.error(error));
 
-    apiCall(null, 'get', `obs?encounter=${id}&includeAll=true&v=full`)
+    apiCall(null, 'get', `obs?encounter=${id}&includeAll=truek&v=full`)
```
Comment (mkiterian) [inline app/js/components/encounters/encountersParent.jsx]: Seems like you have a typo at `truek`

**Comment by Farhanisto** [comment]
Thanks @mkiterian

File: app/js/components/visitsAndEncounters.jsx [hunk 2]
```diff
@@ -234,16 +234,24 @@ export class VisitsAndEncounters extends React.Component {
...
+        className:"filterDate",
+        id: 'date',
+        Header: 'Date',
+        accessor: rowProps => rowProps.date.split("T")[0],  
+        Filter: ({ filter, onFiltersChange}) =>  
+           <DateRangePicker  startDate={this.state.startDate}
+                             endDate={this.state.endDate} ranges={this.state.ranges} 
+                             onEvent={this.handleEvent} 
+                            >
+              <Button className="selected-date-range-btn" style={{width:'100%'}}>
```
Comment (adamsdenniskariuki) [inline app/js/components/visitsAndEncounters.jsx]: declare the `width: '100%'` 'in css since you already have the class `selected-date-range-btn`

**Comment by Farhanisto** [comment]
@adamsdenniskariuki thanks.implemented

**Comment by dkayiwa** [comment]
Can you look into these merge conflicts?

**Comment by Farhanisto** [comment]
@dkayiwa kindly check into this. I have sorted the merge conflict.

**Comment by dkayiwa** [comment]
@Farhanisto can you give me a direct url to module-refapp that shows me these voided observations displayed?

**Comment by Farhanisto** [comment]
@dkayiwa this cannot be shown in the module-refapp. The reason for this is that the .omod file in the ref-app has not Yet been updated. I was afraid of bringing the module-refapp down had I updated it, It however works well on my local machine.

**Comment by dkayiwa** [comment]
Fair point!
I looked at your screenshots on JIRA but never saw any one with voided ones stricken through.

**Comment by Farhanisto** [comment]
@dkayiwa I have added one more screenshot showing this kindly check

**Comment by dkayiwa** [comment]
It should be "Include Deleted" such that we still display the undeleted, but together with the deleted.

**Comment by Farhanisto** [comment]
Should I change the text of the check box from "Show Deleted" to "Include Deleted". @dkayiwa

**Comment by dkayiwa** [comment]
Yes. And the display should have both the deleted and non deleted.

**Comment by Farhanisto** [comment]
@dkayiwa  I have changed the checkbox to "Include Deleted". I have also attached two images on Jira one showing only undeleted observations and the second one showing both deleted and undeleted observations(with a strike-through ) together once the Include Deleted checkbox is clicked. Kindly check

**Pull request merged**